README.md

```diff
@@ -305,6 +319,38 @@ candidates
     └── log.txt
 ```
 
+### Changing query/candidate directory names
+
+In the above examples, DeezyMatch creates `queries` and `candidates` directories, by default, and store the `scenario` (in this example, it is set to `test`) inside these directories. This behaviour can be changed by (see `query_candidate_dirname="my_query_dir"`):
+
+```python
+from DeezyMatch import inference as dm_inference
+
+# generate vectors for queries and candidates
+# note the new argument: query_candidate_dirname 
+dm_inference(input_file_path="./inputs/input_dfm.yaml",
+             dataset_path="dataset/dataset-string-similarity_test.txt", 
+             pretrained_model_path="./models/finetuned_test001/finetuned_test001.model", 
+             pretrained_vocab_path="./models/finetuned_test001/finetuned_test001.vocab",
+             inference_mode="vect",
+             query_candidate_mode="q",
+             query_candidate_dirname="my_query_dir",
+             scenario="test")
+```
+
+In this case, the directory structure is:
+
+```
+my_query_dir/
+└── test
+    ├── embed_my_query_dir
+    ├── input_dfm.yaml
+    ├── log.txt
+    └── my_query_dir.df
+```
+
+(this can also be done for candidates.)
+
```

Comment:
I'm thinking, similarly to [what I said in the comment below](https://github.com/Living-with-machines/DeezyMatch/pull/68#discussion_r463706085), whether it makes sense to have the `query_candidate_mode` option at all, if its function is (correct me if I'm wrong, please) only to specify whether the default parent folder should be either `queries/` or `candidates/`. Especially now that the parent folder can be overwritten. I was wondering whether it would be simpler also for the user if we just specify the full path of the output folder in `scenario`.

It would be like this:
```
dm_inference(input_file_path="./inputs/input_dfm_test50_ft.yaml", dataset_path="inference_candidate_finder/candidate_sets/wikigaz_en.txt", pretrained_model_path="./models/finetuned_test50_trivial_to_ocrgb1900_fc/finetuned_test50_trivial_to_ocrgb1900_fc.model", pretrained_vocab_path="./models/finetuned_test50_trivial_to_ocrgb1900_fc/finetuned_test50_trivial_to_ocrgb1900_fc.vocab", inference_mode="vect", scenario="candidates/feature50test/")
```

instead of this:
```
dm_inference(input_file_path="./inputs/input_dfm_test50_ft.yaml", dataset_path="inference_candidate_finder/candidate_sets/wikigaz_en.txt", pretrained_model_path="./models/finetuned_test50_trivial_to_ocrgb1900_fc/finetuned_test50_trivial_to_ocrgb1900_fc.model", pretrained_vocab_path="./models/finetuned_test50_trivial_to_ocrgb1900_fc/finetuned_test50_trivial_to_ocrgb1900_fc.vocab", inference_mode="vect", query_candidate_mode="c", scenario="feature50test")
```

Let me know what you think. Thanks!!

Reply:
@mcollardanuy Would you please take a look at: https://github.com/Living-with-machines/DeezyMatch/commit/4a45874d18da3bb6d878bc6dfc331b9fd7be038c

I have added a `Quick tour` section in README, so you can check the new syntax/arguments.

Reply:
Hi @kasra-hosseini, that's great, thanks! Both the new structure and the new Quick tour section in the readme 👍